addons/test_website/controllers/portal.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+
+from odoo import http
+from odoo.http import request
+
+
+class PortalTest(http.Controller):
+    """Implements some test portal routes (ex.: for viewing a record)."""
+
+    @http.route('/my/website_test_portal/<int:res_id>', type='http', auth='public', methods=['GET'], website=True)
+    def test_portal_record_view(self, res_id, access_token=None, **kwargs):
+        return request.make_response(f'Record view of test_portal {res_id} ({access_token}, {kwargs})')
```

Comment:
I don't really understand the purpose of this route?

I get that you will use this syntax in your tests, but do you actually need to define the route? It does not look like you "hit" it.